sdk/storage/azure-storage-blobs/inc/azure/storage/blobs/blob_container_client.hpp

```diff
@@ -227,12 +227,13 @@ namespace Azure { namespace Storage { namespace Blobs {
      * @param delimiter This can be used to to traverse a virtual hierarchy of blobs as though it
      * were a file system. The delimiter may be a single character or a string.
      * @param options Optional parameters to execute this function.
-     * @return A ListBlobsByHierarchySegmentResult describing a segment of the blobs in the
+     * @return A ListBlobsByHierarchySinglePageResult describing a segment of the blobs in the
```

Comment:
nit: Are we still using the word `segment` in docs to describe per page?

Reply:
I don't think it's a big deal, it's not in API interface, it's just doc.